src/pasta/fields/fp.rs

```diff
@@ -646,43 +638,43 @@ impl ff::PrimeField for Fp {
 impl FieldExt for Fp {
     const ROOT_OF_UNITY: Self = ROOT_OF_UNITY;
     const ROOT_OF_UNITY_INV: Self = Fp::from_raw([
-        0x9246674078fa45bb,
-        0xd822ebd60888c5ea,
-        0x56d579133a11731f,
-        0x1c88fa9e942120bb,
+        0xf0b87c7db2ce91f6,
+        0x84a0a1d8859f066f,
+        0xb4ed8e647196dad1,
+        0x2cd5282c53116b5c,
```

Comment:
I did not check this constant, but I'm satisfied by its unit test.

Reply:
Checked constants up to here.